host/sgx/windows/enter.asm

```diff
@@ -73,6 +74,14 @@ NESTED_ENTRY oe_enter, _TEXT$00
     mov rax, [rbp+64]
     mov ENCLAVE, rax
 
+    ;; Save the host context in the host stack
+    ;;mov HOST_CONTEXT, rsp
```

Comment:
These comments seem like experiments. Can be removed.

Reply:
Done.